include/net/nrf_cloud_pgps.h

```diff
@@ -239,6 +239,34 @@ int nrf_cloud_pgps_request_all(void);
 #endif /* CONFIG_NRF_CLOUD_MQTT */
 
 #if defined(CONFIG_NRF_CLOUD_PGPS_TRANSPORT_NONE)
+
+/**@brief If REST is not used, but some new transport, call this function
```

Comment:
there is no explicit PGPS REST configuration option is there?  it is either MQTT or NONE for the transport?
do we have any PGPS REST examples?

Reply:
Yes.  I think you added the TRANSPORT_NONE stuff to support REST, but I could be wrong.

The location sample and gnss samples both use REST and support P-GPS with nRF Cloud.